Fails to build with gcc 7

From Debian bug #[853399](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=853399):

> The package fails to build in a test rebuild on at least amd64 with
> gcc-7/g++-7, but succeeds to build with gcc-6/g++-6. The
> severity of this report may be raised before the buster release.
> There is no need to fix this issue in time for the stretch release.
>
> The full build log can be found at:
> http://people.debian.org/~doko/logs/gcc7-20170126/mathicgb_1.0~git20170104-1_unstable_gcc7.log
> The last lines of the build log are at the end of this report.
>
> To build with GCC 7, either set CC=gcc-7 CXX=g++-7 explicitly,
> or install the gcc, g++, gfortran, ... packages from experimental.
>
>   apt-get -t=experimental install g++ 

> Common build failures are new warnings resulting in build failures with
> -Werror turned on, or new/dropped symbols in Debian symbols files.
> For other C/C++ related build failures see the porting guide at
> http://gcc.gnu.org/gcc-7/porting_to.html

    make[3]: Leaving directory '/<<PKGBUILDDIR>>/fflas-ffpack/interfaces'
    Making check in checkers
    make[3]: Entering directory '/<<PKGBUILDDIR>>/fflas-ffpack/checkers'
    make[3]: Nothing to be done for 'check'.
    make[3]: Leaving directory '/<<PKGBUILDDIR>>/fflas-ffpack/checkers'
    make[3]: Entering directory '/<<PKGBUILDDIR>>/fflas-ffpack'
    make[3]: Nothing to be done for 'check-am'.
    make[3]: Leaving directory '/<<PKGBUILDDIR>>/fflas-ffpack'
    make[2]: Leaving directory '/<<PKGBUILDDIR>>/fflas-ffpack'
    Making check in tests
    make[2]: Entering directory '/<<PKGBUILDDIR>>/tests'
    Making check in data
    make[3]: Entering directory '/<<PKGBUILDDIR>>/tests/data'
    make[4]: Entering directory '/<<PKGBUILDDIR>>/tests/data'
    make[4]: Nothing to be done for 'check-am'.
    make[4]: Leaving directory '/<<PKGBUILDDIR>>/tests/data'
    make[3]: Leaving directory '/<<PKGBUILDDIR>>/tests/data'
    make[3]: Entering directory '/<<PKGBUILDDIR>>/tests'
    make  check-TESTS
    make[4]: Entering directory '/<<PKGBUILDDIR>>/tests'
    make[5]: Entering directory '/<<PKGBUILDDIR>>/tests'
    g++ -DHAVE_CONFIG_H -I. -I..  -I.. -g -I../fflas-ffpack/ -I../fflas-ffpack/utils/ -I../fflas-ffpack/fflas/ -I../fflas-ffpack/ffpack -I../fflas-ffpack/field -Wdate-time -D_FORTIFY_SOURCE=2 -O2 -Wall -DNDEBUG -UFFLASFFPACK_DEBUG  -std=gnu++11 -D__FFLASFFPACK_HAVE_CBLAS  -fopenmp  -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -fno-strict-aliasing   -fabi-version=6 -c -o test-lu.o test-lu.C
    In file included from test-lu.C:62:0:
    test-utils.h: In function 'Field* FFPACK::chooseField(Givaro::Integer, uint64_t)':
    test-utils.h:80:24: error: 'bind' is not a member of 'std'
        auto bitrand = std::bind(std::uniform_int_distribution<uint64_t>(2,maxV.bitsize()-1),
                            ^~~~
    test-utils.h:80:24: note: suggested alternative: 'find'
        auto bitrand = std::bind(std::uniform_int_distribution<uint64_t>(2,maxV.bitsize()-1),
                            ^~~~
                            find
    Makefile:1150: recipe for target 'test-lu.o' failed
    make[5]: *** [test-lu.o] Error 1
    make[5]: Leaving directory '/<<PKGBUILDDIR>>/tests'
    Makefile:1393: recipe for target 'check-TESTS' failed
    make[4]: *** [check-TESTS] Error 2
    make[4]: Leaving directory '/<<PKGBUILDDIR>>/tests'
    Makefile:1624: recipe for target 'check-am' failed
    make[3]: *** [check-am] Error 2
    make[3]: Leaving directory '/<<PKGBUILDDIR>>/tests'
    Makefile:1180: recipe for target 'check-recursive' failed
    make[2]: *** [check-recursive] Error 1
    make[2]: Leaving directory '/<<PKGBUILDDIR>>/tests'
    Makefile:572: recipe for target 'check-recursive' failed
    make[1]: *** [check-recursive] Error 1
    make[1]: Leaving directory '/<<PKGBUILDDIR>>'
    dh_auto_test: make -j1 check VERBOSE=1 returned exit code 2
    debian/rules:19: recipe for target 'build' failed
    make: *** [build] Error 2
    dpkg-buildpackage: error: debian/rules build gave error exit status 2

--- a/tests/test-utils.h
+++ b/tests/test-utils.h
@@ -44,6 +44,7 @@
 #include <givaro/givranditer.h>
 #include <chrono>
 #include <random>
+#include <functional>
 
 namespace FFPACK {
 

